bin/sdr

```diff
@@ -11,6 +11,21 @@ class CLI < Thor
     true
   end
 
+  no_commands do
+    def check_control_master(skip)
+      unless skip
+        say("Checking SSH connections for #{options[:environment]} environment requires a control master.")
+        control_master = ask('Have you started a control master session: (y/n): ')
+        exit unless control_master.downcase == 'y'
+      end
+
+      return if `ps aux` =~ /ControlMaster=yes/
+
+      say('Control master not detected. Please start a control master session.')
+      exit
+    end
```

Comment:
I was thinking that an initial approach might be to use the `tty-prompt` library (already in `sdr-deploy`) to prompt the developer asking if they have a CM going already. If we're going to check for the CM ourselves, though, I'm not sure what the prompt gets us. Here's more or less what I had in mind, pseudocode-style:

```
Was skip flag provided?
* Yes. Do nothing.
* No.
  * Check for active CM. Is it present?
    * Yes. Do nothing.
    * No. Complain and abort.
```

Reply:
Yeah, that works too. I was thinking the prompt just saves them rerunning the app, but that's a trivial savings for sure.